plasmapy/__init__.py

```diff
@@ -33,3 +33,26 @@ class UnsupportedPythonError(Exception):
     from . import physics
     from . import utils
 
+
+def online_help(query):
+    """
+    Search the online PlasmaPy documentation for the given query from plasmapy.org
+    Opens the results in the default web browser.
+    Requires an active Internet connection.
+    Redirects to Astropy.units in case of query 'unit' or 'units'
+
+    Parameters
+    ----------
+    query : str
+        The search query.
+    """
+    from urllib.parse import urlencode
+    import webbrowser
+
+    url = 'http://docs.plasmapy.org/en/stable/search.html?\
+    {0}&check_keywords=yes&area=default'.format(urlencode({'q': query}))
+
+    if((query.lower() == 'units') or (query.lower() == 'unit')):
```

Comment:
This looks nice to me but @namurphy and others would probably give the final call if this check is needed. You could write it something like this:
```python
...
query_lower = query.lower()
if query_lower == 'units' or query_lower == 'unit'
    ...
```
to avoid compute `query.lower()` twice. :smile:

Reply:
Another way to do it (if you don't need to use `query_lower` ever again)
```Python
if query.lower() in ['units', 'unit']:
  ...
```

Reply:
That's even better! We most likely want to use a tuple instead of a list though (just a bit faster) `('units', 'unit')`.